worker/uniter/uniter.go

```diff
@@ -295,8 +295,8 @@ func (u *Uniter) loop(unitTag names.UnitTag) (err error) {
 			errorString = err.Error()
 		}
 		if errors.Cause(err) == ErrCAASUnitDead {
+			errorString = err.Error()
```

Comment:
The error string is "unit dead", which is what the free-form string would have been changed to. Might as well ditch the custom string.